client/llb/source.go

```diff
@@ -249,7 +249,7 @@ const (
 //
 // By default the git repository is cloned with `--depth=1` to reduce the amount of data downloaded.
 // Additionally the ".git" directory is removed after the clone, you can keep ith with the [KeepGitDir] [GitOption].
-func Git(url, ref string, opts ...GitOption) State {
+func Git(url, fragment string, opts ...GitOption) State {
```

Comment:
`fragment` needs explanation

Reply:
Note that there is no functional change in here. The previous var was incorrectly named; it was actually a `ref:substring` pair.